### DISCUSSION TITLE
Family Accounts

### DISCUSSION THREAD

**Discussion by Rom0067** [ellite/Wallos#528] [Ideas]
I would love to see this app support a master family account.  What I mean is, as a parent in a family of four, I would like to have the ability where all users can have their own individual account but can be viewed from the master/parent/admin account.  It would make managing all of our subscriptions easy.  My wife and i could collaborate and organize our finances.  Also I think it would allow for great teachable moments with the kids.   

Right now me and my wife sign into 1 account and it works pretty well but having the ability to do the above would be amazing.  All in all the app is great and we look forward to using it.

**Comment by tismofied** [comment]
I fully support this feature. As a parent in a family of four as well, I would love  if I have the option to keep my account separate from my wifes' so I can show her on her account where most of my money is going lol.
